src/bubble_chart/bubble_chart.js

```diff
@@ -205,10 +206,10 @@ class BubbleChart extends Component {
     node
       .on("mousemove", function (d) {
         d3.select("#chart").append("div").attr("id", "tooltip");
-
         let tooltip_html = "";
         tooltip_html += "<div><span>" + d.data.itemName + "<br/></span>";
-        tooltip_html += "<span>   " + d.data.rendered + "</span></div>";
+        tooltip_html += "<span>   " + (d.data.html ? DOMPurify.sanitize(d.data.html) : d.data.rendered) + "<br/></span>";
+        tooltip_html += "<span>   " + (d.data.secondMeasureHtml ? DOMPurify.sanitize(d.data.secondMeasureHtml) : d.data.secondmeasureRendered) + "</span></div>";
```

Comment:
`secondmeasureRendered`

Lower case 'm' for measure?

`secondMeasureHtml` has upper case 'M'

Reply:
Ok. Updated this in both places. Thank you!